beps/con-sap.jco/1444_explicit_parameter_categories_in_sap_jco_execute.md

```
@@ -0,0 +1,335 @@
+    RfcParameters parameters = {},
+    typedesc<RfcRecord|xml|json?> returnType = <>
+) returns returnType|Error = @java:Method {
+    'class: "io.ballerina.lib.sap.Client"
+} external;
+```
+
+Key changes from the current signature:
+
+| Aspect                     | Before                                     | After                                           |
+|----------------------------|--------------------------------------------|-------------------------------------------------|
+| Input type                 | `record {                                  | FieldType?...;                                  |}` (flat, category-free) | `RfcParameters` (explicit sections) |
```

markdownlint-cli2 (0.22.0)

[warning] Table column count
Expected: 3; Actual: 5; Too many cells, extra data will be missing

(MD056, table-column-count)